modules/utils/validation.js

```diff
@@ -114,10 +120,11 @@ function validateRule (item, path, itemId, meta, c) {
     logger.warn(`No config for field ${field}`);
     field = null;
   }
-  if (field == null) {
-    properties = ["operator", "operatorOptions", "valueSrc", "value"].reduce((map, key) => map.delete(key), properties);
-    operator = null;
-  }
+  // TODO PZ: fix validation
```

Comment:
Do the TODO?

Reply:
I think we don't need to fix validation in this case. Since LHS can be a value or a function, field can be null and we don't want to clear operator. I think the purpose of it was only to check the model before displaying the query, because it is not possible to set field to null once it has some value selected